src/stk/calculators/calculator.py

```diff
@@ -0,0 +1,84 @@
+"""
+Calculator
+==========
+
+
+
+"""
+
+
+class Calculator:
+    def __init__(self, use_cache):
+        self._use_cache = use_cache
+        self._cache = {}
+
+    def set_cache_use(self, use_cache):
+        """
+        Set cache use on or off.
+
+        Parameters
+        ----------
+        use_cache : :class:`bool`
+            ``True`` if the cache is to be used.
+
+        Returns
+        -------
+        :class:`.Calculator`
+            The calculator.
+
+        """
+
+        self._use_cache = use_cache
+
+    def is_caching(self):
```

Comment:
This is entirely my own opinion but I think this function name is slightly unclear. I think they should represent an action taken on the class, e.g. ```def check_caching(self):``` .
```is_caching``` sounds more like a variable name because it sounds like a state of something rather than a function.
As a user I would be inclined to use this as 
```python 
calculator.is_caching 
```
rather than 
```python 
calculator.is_caching() 
```

Reply:
The general convention in stk is that a method which returns a bool value begins with is_.  The reason for this is that when you write an if statement it is easy to read. for example

```python
if calculator.is_caching():
    # do stuff
```

So I think this needs to be addressed in a separete github issue - ie file an issue for changing the convention.